src/main_loops/src/client_loop.cpp

```diff
@@ -53,6 +65,13 @@ int main(int argc, char * argv[]) {
 	glfwSetKeyCallback(mWindow, key_callback);
 	glfwSetMouseButtonCallback(mWindow, mouse_click_callback);
 
+    ShaderLoader loader;
+
+    projection = glm::perspective(45.0f, (float)window_width / (float)window_height, 0.1f, 1000.0f);
+    view = glm::lookAt(cam_pos, cam_look_at, cam_up);
+
+    Model m(loader, "../models/cube/cube.obj", "../src/graphics/shaders/playershader.vert", "../src/graphics/shaders/playershader.frag");
```

Comment:
Maybe we can have a file to declare paths or make configurable. As long as the paths are defined somewhere. Would help if we have a lot of models

Reply:
I agree, I think this is something that can be done with #5 ?